Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/discover/Discover.kt

```diff
@@ -38,23 +38,19 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import com.example.jetcaster.R
 import com.example.jetcaster.core.data.database.model.Category
+import com.example.jetcaster.core.data.model.FilterableCategory
+import com.example.jetcaster.core.data.model.PodcastCategoryFilterResult
 import com.example.jetcaster.designsystem.theme.Keyline1
-import com.example.jetcaster.ui.home.category.PodcastCategoryViewState
 import com.example.jetcaster.ui.home.category.podcastCategory
 
-data class DiscoverViewState(
-    val categories: List<Category> = emptyList(),
-    val selectedCategory: Category? = null
-)
-
 fun LazyListScope.discoverItems(
-    discoverViewState: DiscoverViewState,
-    podcastCategoryViewState: PodcastCategoryViewState,
+    filterableCategories: List<FilterableCategory>,
+    podcastCategoryFilterResult: PodcastCategoryFilterResult,
```

Comment:
should we consider enabling strong skipping mode @bentrengrove ?

Reply:
Do we demonstrate this right now in a sample? Definitely open to enabling strong skipping mode!

Reply:
It's not in any samples yet, you are welcome to add it. I am planning on doing it in the future to all samples (or it will just be switched on by default)